### ISSUE TITLE
better describe linkage with git-evtag

### ISSUE THREAD

**Issue by cgwalters** [indutny/git-secure-tag#1]
This is now basically a JS reimplementation, right?  In that case, can we make that clearer, and link to git-evtag a bit more prominently in the intro?

Calling this `git-evtag-js` might be clearer for example.  And given that I think no one would really feel a need to parallel install them, why not just take the `git evtag` name, i.e. install the binary as `/usr/bin/git-evtag`.

I'm happy to cross link from git-evtag (which is now the C, Python, and Rust implementation) too.

**Comment by indutny** [comment]
Of course! Could you please open a PR that will changes the README in desired way?

Regarding name, I can certainly make it install `git-evtag` binary as well, but I don't really like renaming this repo and/or npm module. IMO, `secure-tag` conveys the goal of the project in much better way. What do you think?

**Comment by zbeekman** [comment]
This issue definitely needs to be resolved ASAP. As a user I tried to install cgwalters/git-evtag but ran into issues compiling (on MacOS Yosemite). It is not clear to me what differences there are, if any, between the two tools, other than naming conventions.

**Comment by indutny** [comment]
@zbeekman the tools are completely compatible, it is just the name that differs.
